src/test/util_sds_check/out3.log.match

```diff
@@ -0,0 +1,7 @@
+util_sds_check$(nW)TEST3: START: util_sds_check
+ $(nW)util_sds_check test_invalid_uuid
+src version: $(nW)
+compiled with support for shutdown state
+compiled with libndctl $(nW)
+an ADR failure was detected but the pool was closed - SDS will be reinitialized
```

Comment:
The UUID has changed. So, most probably, the pool has been moved. Why we call it an "ADR failure"? 🤣 Reinit part is ok. But IMHO we should adjust the warning message to differentiate between USC mismatch and UUID mismatch. 

```suggestion
the pool has moved to a new location but it was closed properly - SDS will be reinitialized
```

Reply:
Done.